src/ParallelAlgorithms/NonlinearSolver/NewtonRaphson/ElementActions.hpp

```diff
@@ -323,10 +324,11 @@ struct PerformStep {
     if (UNLIKELY(get<logging::Tags::Verbosity<OptionsGroup>>(box) >=
                  ::Verbosity::Debug)) {
       Parallel::printf(
-          "%s %s(%zu): Perform step with length %f\n", array_index,
+          "%s %s(%zu): Perform step with length: %s\n", array_index,
           Options::name<OptionsGroup>(),
           db::get<Convergence::Tags::IterationId<OptionsGroup>>(box),
-          db::get<NonlinearSolver::Tags::StepLength<OptionsGroup>>(box));
+          get_output(
+              db::get<NonlinearSolver::Tags::StepLength<OptionsGroup>>(box)));
```

Comment:
I'm pretty sure this formats like "%g", maybe with a precision indicated.

Reply:
Thanks, pushed another fixup